Dockerfile

```diff
@@ -1,5 +1,6 @@
 # ================ PYTHON
-FROM python:3.9.7
+ARG PYTHON_VERSION=3.9
```

Comment:
I assume this sets a default, but in practice is always overridden by the matrix setting?

Reply:
That's correct 👍